sklearn/linear_model/base.py

```diff
@@ -436,14 +436,14 @@ def fit(self, X, y, sample_weight=None):
                 sample_weight).ndim > 1):
             sample_weight = column_or_1d(sample_weight, warn=True)
 
-        X, y, X_mean, y_mean, X_std = self._center_data(
-            X, y, self.fit_intercept, self.normalize, self.copy_X,
-            sample_weight=sample_weight)
-
         if sample_weight is not None:
             # Sample weight can be implemented via a simple rescaling.
             X, y = _rescale_data(X, y, sample_weight)
 
+        X, y, X_mean, y_mean, X_std = self._center_data(
+            X, y, self.fit_intercept, self.normalize, self.copy_X,
+            sample_weight=None)
```

Comment:
@Sandy4321 we are working on solving the issue at the moment. In the meantime, if you want to use sample weights, you can instead multiply `X`(row by row) and `y` with the square root of the weights, as it is done in the code above. Fitting `LinearRegression` without sample weights still runs without issues.